### PR TITLE
Some spring cleaning in the repo

### PR DESCRIPTION
**Pull request by orta**
Mainly cleaning up some docs, adding the dependencies issue template, fixing some TODOs and updating some infra.

### CODE REVIEW

File: dangerfile.ts [hunk 1]
```diff
@@ -7,6 +7,19 @@ const markdowns = files.filter(f => f.indexOf(".md") !== -1)
...
+  { word: "Github", reason: "Please use GitHub, capital 'H'" },
+  { word: "Cocoapods", reason: "Please use CocoaPods, capital 'P'" },
+  { word: "Javascript", reason: "Please use JavaScript, capital 'S'" },
+  { word: "Typescript", reason: "Please use TypeScript, capital 'S'" },
+  { word: "Fastlane", reason: "Please use fastlane, lowercase 'f'" },
+  { word: "localhost:4000", reason: "You may have left an internal link in the markdown" },
+  { word: "[]: ???", reason: "You've missed a link" },
+  { word: "[TODO]", reason: "You may have missed a TODO here" },
+  { word: "react native", reason: "Please use React Native with capitals" }
+]
```
Comment (ashfurrow) [inline dangerfile.ts]: Nice.

File: playbooks/informationals.md [hunk 2]
```diff
@@ -18,11 +18,28 @@ If you're going to do it on-site in NYC, ask them to met you at Artsy, and give
...
+them a link to this doc.
+
+#### Preparation
+
+For the person on the other end of the informational this is a big event. You may do these for Artsy with
+regularity, so it's important to always take time to do research on the person you're talking to and not get
+complacent that you can just sell Artsy.
+
+- Take the time to read through their CV
+- Google them, see what their public internet presence is
```
Comment (ashfurrow) [inline playbooks/informationals.md]: We should specify this is to set the discussion context, and not for evaluating the candidate.

File: practices/history.md [hunk 3]
```diff
@@ -3,9 +3,61 @@ title: The history of practices at Artsy
...
+- Platform
+- Mobile
+
+The product teams [simplified](https://groups.google.com/a/artsymail.com/d/msg/team/hwLvMwuBmJo/8xGcssucBwAJ) 🔒
+this down into 4 major teams at the start of 2016:
+
+- Auctions
+- Collector Experience & GMV
+- Partner Success & Revenue
+- Publishing
```
Comment (ashfurrow) [inline practices/history.md]: I believe we had a Practice team as well, or at least I remember hearing their updates at Standup.

Comment (orta) [reply practices/history.md]: The Platform team? - yeah, I scoped it to product teams from that announcement emails

**Comment by orta** [comment]
OK, addressed, thanks - merging

**Pull request merged**